draft-ietf-httpbis-priority.md

```diff
@@ -52,12 +52,12 @@ by the retrieval of a CSS file that the document refers to.  In contrast, inline
 images do not block rendering and get drawn incrementally as the chunks of the
 images arrive.
 
-To provide meaningful representation of a document at the earliest moment, it is
+To provide meaningful presentation of a document at the earliest moment, it is
```

Comment:
`representation` is an http keyword. I'd use "presentation" or something similar.